chat.js

```diff
@@ -156,8 +156,23 @@ Chat.namefilter = function (name, user) {
 		// \u534d\u5350 swastika
 		// \u2a0d crossed integral (f)
 		name = name.replace(/[\u00a1\u2580-\u2590\u25A0\u25Ac\u25AE\u25B0\u2a0d\u534d\u5350]/g, '');
+		name = name.replace(/[\u2800]+/g, ' ').trim();
```

Comment:
What is this? It probably needs a comment.

Reply:
idk actually but it was in the code you provided so I figured you would know.